internal/brokers/manager.go

```diff
@@ -51,8 +51,7 @@ func NewManager(ctx context.Context, configuredBrokers []string, args ...Option)
 
 	// Set default options.
 	opts := options{
-		rootDir:      "/",
-		brokerCfgDir: "etc/authd/broker.d",
+		brokerCfgDir: consts.DefaultBrokerCfgDir,
```

Comment:
That's not really the case (saving directories in `golden/`). I knew we reverted this in the other PR, but I thought it was to avoid the chaining changes in other packages (when doing the brokers tests). Now that I actually touched those other packages (even though now you've mentioned that I shouldn't have), I decided to switch this because it also affects broker discovery and so on (when creating the service, etc.). It's annoying that it chains the `WithOptions` indeed, but we have something a bit similar in `adsys`, so I thought it would be ok.